kb/development/mbedtls-coding-standards.md

```diff
@@ -210,18 +268,17 @@ Most functions should return `int`, more specifically `0` on success (the operat
 
 Function should avoid in-out parameters for length (multiplexing buffer size on entry with length used/written on exit) since they tend to impair readability. For example:
 ```c
-    mbedtls_write_thing( void *thing, unsigned char *buf, size_t *len ); // no
-    mbedtls_write_thing( void *thing, unsigned char *buf, size_t buflen,
-                         size_t *outlen ); // yes
+mbedtls_write_thing(..., unsigned char *buf, size_t *len); // no
+mbedtls_write_thing(..., unsigned char *buf, size_t buflen, size_t *outlen); // yes
 ```
 
 For PSA functions, [input buffers](https://armmbed.github.io/mbed-crypto/html/overview/conventions.html#input-buffer-sizes) have a `size_t xxx_size` parameter after the buffer pointer, and [output buffers](https://armmbed.github.io/mbed-crypto/html/overview/conventions.html#output-buffer-sizes) have a `size_t xxx_size` parameter for the buffer size followed by a `size_t *xxx_length` parameter for the output length. This convention is also preferred in new `mbedtls_xxx` code, but older modules often use different conventions.
```

Comment:
Can we link to the canonical PSA API pages instead? But maybe that should be a different PR...

The 1.1 specification is at https://arm-software.github.io/psa-api/crypto/1.1/

Reply:
any update on having a `latest` link?